cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -1174,6 +1216,19 @@
                     else:
                         query_new_status = 'submitted'
                         job.set_submitted()
+                        mail_sending_job_submitted = self.app.config.get('conf').mail_sending_job_submitted
+                        # send submitted email
+                        if mail_sending_job_submitted:
+                            try:
+                                time_request = self.par_dic.get('time_request', '')
+                                self.send_email('submitted',
+                                                instrument=self.instrument.name,
+                                                time_request=time_request)
+                                # store an additional information about the sent email
+                                query_out.set_status_field('email_status', 'email sent')
+                            except MailNotSent as e:
```

Comment:
we should be sure the problem is sent to sentry. we have to be certain to react on this

Reply:
I will need some clarification on this